src/components/ChangePlan/CheckoutProcess.js

```diff
@@ -0,0 +1,361 @@
+import React, { useEffect, useState } from 'react';
+import { InjectAppServices } from '../../services/pure-di';
+import HeaderSection from '../shared/HeaderSection/HeaderSection';
+import { useIntl } from 'react-intl';
+import { useRouteMatch } from 'react-router-dom';
+import { Loading } from '../Loading/Loading';
+import {
+  FieldGroup,
+  FieldItem,
+  FormWithCaptcha,
+  InputFieldItem,
+  PhoneFieldItem,
+  SelectFieldItem,
+  SubmitButton,
+} from '../form-helpers/form-helpers';
+import { Helmet } from 'react-helmet';
+
+const ContactInformation = ({
```

Comment:
Perhaps it would be nice to differentiate this section and the other 2 when they are done, inside the checkout, something like this:
- ContactInformationSection
- ContactInformationStep
...

So then inside the checkout component you would read something like this:
```
<ContactInformationStep>
<BillingStep>
<PaymentStep>
```
What do you think? does it make sense?

Reply:
you idea is use the name of the component:

``` 
<ContactInformationStep> 
<BillingInformationStep>
<PaymentStep>
```

instead of:

```
<ContactInformation>
<BillingInformation>
<Payment>
```

Is this right?

Reply:
Also this component could be in a different file, to help readability.

Reply:
Pensandolo bien, podemos querer tener estas secciones por separado para rehusar, queda mejor como lo propusiste sin Step ni Section, viste que en Doppler tenemos la seccion de informacion de contacto en Panel de Control como algo aparte. Y la seccion de billing tambien como algo aparte, por si queres cambiar algo en algun momento.

Reply:
Se creo un componente separado para ContactInformation con la idea de poder reutilizarlo